### PR TITLE
Added MainContent and SideBar with desktop styling

### PR DESCRIPTION
**Pull request by irfanuddinahmad**
Description: This PR adds the main content area and sidebar area as the children of the Layout component for the desktop layout

JIRA: https://openedx.atlassian.net/browse/ENT-1840 and 
https://openedx.atlassian.net/browse/ENT-1919

### CODE REVIEW

File: src/components/Content/SideBar.jsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      href: 'https://edx.org',
+    },
+    {
+      title: 'Student Record',
+      href: 'https://edx.org',
+    },
+  ],
+};
+
+class SideBar extends Component {
```
Comment (adamstankiewicz) [inline src/components/Content/SideBar.jsx]: nit: rename to `Sidebar`

File: src/components/Content/SideBar.jsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  state = {};
+
+  render() {
```
Comment (adamstankiewicz) [inline src/components/Content/SideBar.jsx]:
```jsx
render() {
  return (
    <div className="col">
      <Links />
    </div>
  );
}
```

File: src/components/Content/SideBar.jsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    },
+    {
+      title: 'Student Record',
+      href: 'https://edx.org',
+    },
+  ],
+};
+
+class SideBar extends Component {
+  state = {};
```
Comment (adamstankiewicz) [inline src/components/Content/SideBar.jsx]: State is unused in this component, so no need to include it here.

File: src/components/Content/SideBar.jsx [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    );
+  }
+}
+
+SideBar.defaultProps = {
+  links: [],
+};
+
+SideBar.propTypes = {
+  links: PropTypes.arrayOf(PropTypes.shape({
```
Comment (adamstankiewicz) [inline src/components/Content/SideBar.jsx]: The `Links` component is getting its data from a local variable, so no need for the `propTypes` definition of `links` here. Likewise for `defaultProps`.

File: package.json [hunk 5]
```diff
@@ -39,18 +39,18 @@
     "redux-thunk": "^2.2.0"
   },
   "devDependencies": {
+    "autoprefixer": "^9.4.5",
```
[the remark this answers is not in the document]
Comment (irfanuddinahmad) [reply package.json]: Seem to have become sorted ...

File: src/components/Content/SideBar.jsx (hunk 1 above)
Comment (irfanuddinahmad) [reply src/components/Content/SideBar.jsx]: Done

File: src/components/Content/SideBar.jsx (hunk 3 above)
Comment (irfanuddinahmad) [reply src/components/Content/SideBar.jsx]: Removed

File: src/components/Content/SideBar.jsx (hunk 4 above)
Comment (irfanuddinahmad) [reply src/components/Content/SideBar.jsx]: Removed

File: src/components/Content/MainContent.jsx [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+        completed: courseRuns.filter(courseRun => courseRun.status === 'completed'),
+      },
+    });
+  }
+
+  render() {
+    const { courses, isProgramEnrollmentsLoading } = this.state;
+
+    return (
+      <React.Fragment>
```
[the remark this answers is not in the document]
Comment (irfanuddinahmad) [reply src/components/Content/MainContent.jsx]: Done

File: src/components/DashboardHome/DashboardHome.jsx [hunk 7]
```diff
@@ -1,103 +1,33 @@
 import React from 'react';
 
 import Hero from '../Hero/Hero';
-import CourseSection from '../CourseSection/CourseSection';
-import InProgressCourseCard from '../CourseCard/InProgressCourseCard';
-import UpcomingCourseCard from '../CourseCard/UpcomingCourseCard';
-import CompletedCourseCard from '../CourseCard/CompletedCourseCard';
-
-import sampleApiResponse from './sampleApiResponse';
+import MainContent from '../Content/MainContent';
```
[the remark this answers is not in the document]
Comment (irfanuddinahmad) [reply src/components/DashboardHome/DashboardHome.jsx]: Done

File: src/components/Content/Sidebar.jsx [hunk 8]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+import Links from '../Links/Links';
+import linksData from './sampleLinks';
+
+const Sidebar = () => (
+  <div className="ml-5">
```
[the remark this answers is not in the document]
Comment (irfanuddinahmad) [reply src/components/Content/Sidebar.jsx]: Done

File: src/components/Content/sampleLinks.js [hunk 9]
```diff
@@ -0,0 +1,24 @@
+/*
+ * The `Sidebar` component will likely handle the fetching of
```
[the remark this answers is not in the document]
Comment (irfanuddinahmad) [reply src/components/Content/sampleLinks.js]: Done

File: src/components/DashboardHome/Sidebar.jsx [hunk 10]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+import Links from '../Links/Links';
+import linksData from './sampleLinks';
+
+const Sidebar = () => (
+  <div>
```
Comment (mushtaqak) [inline src/components/DashboardHome/Sidebar.jsx]: we can use `<>` instead of empty div.

Comment (irfanuddinahmad) [reply src/components/DashboardHome/Sidebar.jsx]: Done

**Pull request merged**